Userland/Utilities/uptime.cpp

```diff
@@ -27,7 +29,7 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
 
     auto file = TRY(Core::File::open("/sys/kernel/uptime"sv, Core::File::OpenMode::Read));
 
-    TRY(Core::System::pledge("stdio"));
+    // TRY(Core::System::pledge("stdio"));
```

Comment:
Please don't commit commented out code like this

Reply:
Yeah sure, that prevents people from wasting time on reviewing something that isn't ready for review.

Reply:
Sorry, did the review policy change? I thought you had to request someone to review.